config/ksql-server.properties

```diff
@@ -14,3 +14,6 @@
 
 bootstrap.servers=localhost:9092
 listeners=http://localhost:8088
+ksql.processing.log.stream.auto.create=true
```

Comment:
need to update these (and the other uses) to the new prefix (logging.processing)